azure-pipelines.yml

```diff
@@ -21,6 +13,25 @@ jobs:
   steps:
   - template: azure-pipelines-steps.yml
 
+  - script: npm pack
+    displayName: 'npm pack'
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+
+  - task: CopyFiles@2
```

Comment:
Do we need this step?

Reply:
Fixed. Sorry meant to publish this as draft :)

Reply:
Can't you just point directly at the tgz in the publishBuildArtifacts step?

Reply:
To be clear, this is also fine